l10n_ch_scan_bvr/wizard/scan_bvr.py

```diff
@@ -353,8 +353,9 @@ def validate_bvr_string(self):
             domain = [('ccp', '=', data['bvr_struct']['beneficiaire'])]
         else:
             domain = \
-                [('ccp', '=', data['bvr_struct']['beneficiaire']),
-                 ('bvr_adherent_num', '=', data['bvr_struct']['bvrnumber'])]
+                [('ccp', '=', data['bvr_struct']['beneficiaire']), '|',
+                 ('bvr_adherent_num', '=', data['bvr_struct']['bvrnumber']),
+                 ('bvr_adherent_num', '=', False)]
```

Comment:
@Martronic-SA Shouldn't we refactor it to first search for a matching `bvr_adherent_num` then if not found search for a `bvr_adherent_num` set to false?

Plus, what if the ccp field is present on multiple accounts in your ERP? (with Bank's CCP)

You could end up matching the wrong supplier.

Reply:
This has been resolved by the commit https://github.com/OCA/l10n-switzerland/pull/447/commits/a8a8d316f5730525897a20a414169e23b738b6d9 below.